### PR TITLE
Move Training infrastructure documentation from cloud section to training section

### PR DESCRIPTION
**Pull request by gwarf**
Move Training infrastructure documentation from cloud section to training section, as requested by @glarocca.

### CODE REVIEW

File: content/en/users/training/infrastructure/_index.md [hunk 1]
```diff
@@ -1,20 +1,166 @@
...
 description: >
-  Training infrastructure
+  The training infrastructure on EGI Cloud 
 ---
 
-{{% pageinfo %}}
-This is a placeholder page.
-{{% /pageinfo %}}
+The training infrastructure is a resource pool within the EGI Federated
+Cloud infrascture providing IaaS as well ass access services (login,
```
Comment (enolfc) [inline content/en/users/training/infrastructure/_index.md:10]:
```suggestion
Cloud infrascture providing IaaS as well as access services (login,
```

Comment (gwarf) [reply content/en/users/training/infrastructure/_index.md:10]: Duh, missed your comment, we will keep an ass in the doc for some more time ;)

Comment (gwarf) [reply content/en/users/training/infrastructure/_index.md:10]: Once the code owner PR will be merged I will make another one to fix this and see if codeowners conf is working.

Comment (gwarf) [reply content/en/users/training/infrastructure/_index.md:10]: Done in #45

**Pull request merged**